src/acceptance-test/java/org/zalando/nakadi/webservice/EventStreamReadingAT.java

```diff
@@ -252,7 +252,7 @@ public void whenGetEventsWithUknownTopicThenTopicNotFound() {
                 .then()
                 .statusCode(HttpStatus.NOT_FOUND.value())
                 .and()
-                .contentType(equalTo("application/problem+json;charset=UTF-8"))
```

Comment:
For some reason (I need to investigate), upgrading spring boot meant that `charset=UTF-8` is not returned anymore. I don't think the change makes a difference for clients.